tasks/main.yml

```diff
@@ -13,9 +13,26 @@
 - name: Install FreeIPA client
   block:
     - name: Install FreeIPA client (Debian Bullseye)
-      ansible.builtin.package:
-        name: "{{ package_names }}"
-        default_release: "{{ ansible_distribution_release }}-backports"
+      block:
+        - name: Install FreeIPA client (Debian Bullseye)
+          ansible.builtin.package:
+            name: "{{ package_names }}"
+            default_release: "{{ ansible_distribution_release }}-backports"
+        # On Debian Bullseye, the default /etc/krb5.conf file uses a
+        # FILE value for this option.  The /etc/krb5.conf file is then
+        # overwritten when the 00_setup_freeipa.sh script is run, and
+        # the new file uses the KEYRING value specified below.  This
+        # causes a problem since before running that script we kinit,
+        # which stores our keytab in a file that is then ignored by
+        # Kerberos due to the new configuration.  To avoid this
+        # wrinkle we simply adjust the default configuration to use
+        # the preferred value before the script can ever be run.
+        - name: Set default ccache name for Kerberos (Debian Bullseye)
+          community.general.ini_file:
```

Comment:
I think this needs a `mode` option for completeness and to quell the Ansible-lint annotations.

Reply:
Please see commit df66589.